domains/runtime/evm/Cargo.toml

```diff
@@ -21,6 +21,7 @@ codec = { package = "parity-scale-codec", version = "3.6.12", default-features =
 domain-check-weight = { version = "0.1.0", path = "../../pallets/domain-check-weight", default-features = false }
 domain-pallet-executive = { version = "0.1.0", path = "../../pallets/executive", default-features = false }
 domain-runtime-primitives = { version = "0.1.0", path = "../../primitives/runtime", default-features = false }
+# just here to activate the serde feature
```

Comment:
can't we just enable this feature through the primitives here ?
The comment is confusing without context

Reply:
I tried that yesterday and it didn’t work. I’ll have another go today.

Reply:
Well, now it works 🤷